app/models/review.rb

```diff
@@ -1,4 +1,10 @@
 class Review < ApplicationRecord
   belongs_to :user, class_name: 'User'
   belongs_to :writer, class_name: 'User'
+
+  validates :content, presence: true
+  validates :rating, presence: true
+  validates :rating, inclusion: { in: [0, 1, 2, 3, 4, 5] }
```

Comment:
Can we make this from 1 to 5? Since we have the stars for the rating we need them to be more than 0

Reply:
fixed!